iotdb/src/main/java/org/apache/iotdb/db/engine/cache/LruLinkedHashMap.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.engine.cache;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * This class is an LRU cache. <b>Note: It's not thread safe.</b>
+ */
+public abstract class LruLinkedHashMap<K, V> extends LinkedHashMap<K, V> {
```

Comment:
```suggestion
public abstract class LRULinkedHashMap<K, V> extends LinkedHashMap<K, V> {
```

Reply:
ok

Reply:
@qiaojialin actually I don't like the function `suggested change`. Look here, you changed the class name but forget to change the file name, which Suyue doesn't notice too.
I personally prefer you just point out where the problem is instead of changing it for her.